e2e/eslint.config.mjs

```
@@ -0,0 +1,26 @@
+export default defineConfig([
+  { ignores: ["node_modules", "playwright-report", "test-results"] },
+  tseslint.configs.recommended,
+  js.configs.recommended,
+  {
+    files: ["**/*.{js,ts}"],
+    languageOptions: { globals: { ...globals.browser, ...globals.node } },
+    rules: {
+      "no-unused-vars": "off",
+      "@typescript-eslint/no-unused-vars": [
+        "error",
+        { varsIgnorePattern: "^_", varsIgnorePattern: "^_" },
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)